### DISCUSSION TITLE
[Design/Balance] Hal's Request: All In Slaughter Demon Rework

### DISCUSSION THREAD

**Discussion by Rurikz** [ParadiseSS13/Paradise#22430] [Documents]
You want an all in change on Slaughter? Have the best I can offer

**Objective**: 
    • Keep the major midround roll.
    • Keep the low 1% of rolling
    • Remove hit and run reliance
    • Raise skill floor so newbie slaughter doesnt die in 5 seconds flat
    • More interesting and engaging for crew to fight, instead of them throwing their hands in the air and salting.
    • Keeping current idea of it being the lowest chance to roll, this is meant to be kept as the _HIGHEST THREAT MIDROUND_, like it currently it is. Killing it should require cooperation from the entire crew, just like vs xenos/terrors/blob.

**Damage and Stacking Tweaks**: 
    • HP from 130 > 400.
    • Damage remains the same, 30.
    • Armor pen maxed, so a miner will be killed just as fast as an assistant.
    • Object damage raised high enough to break walls. Should also be able to break windows in one hit.
    • Stacking damage resist and additional damage per hit on ckey controlled mob, 5% for both each hit, capped at 50%, starts lowering after 5 seconds without hitting anybody, should reach 0 stacks in 10 seconds.
    • Every melee hit against ckey controlled mob heals 10 HP. They must be alive for it to heal.
    • Every melee hit refreshes the temporary speed gained from surfacing from blood crawl, resetting the timer till it reverts to the normal, slower demon speed.

**Bloodcrawl Changes**:
    • Bloodcrawl cooldown is set to 20 seconds. 
    • Two second delay where you have to still to “channel” into blood. This ideally comes with a sprite effect to notify this is happening.
    • In tandem with the above, demon becomes unable to be pushed or shuffled.
    • Passive heal at the rate of 5% per second while in blood nullspace. This is capped at 200 HP.
    • Healing is preformed differently. It is no longer through dragging a corpse and bloodcrawling. Instead, when channeling is completed with Bloodcrawl, **ALL** corpses in a 3x3 square are taken into blood. 
    • In tandem with the above, each ckey controlled corpse heals 100 HP, and increases max HP by 10. If four corpses or more are eaten at once, you get an additional 50 max HP increase, atop of the flat 10 values.
    • When Bloodcrawling out of nullspace, the blood within a 5x5 square begins to glow. The channeling will not be cancelled unless all blood within that square is cleaned. Any humanoids within the radius when the demon’s channeling is completed will be knocked down and start bleeding.

**Additional Abilities**:
    • Blood Charge: Smashes through walls, obstacles, and people, in a similar fashion to rod form. I imagine it having 7 tile range, unable to change direction once chosen, critting people if it hits them head on. Ideally, it could be given some sort of scream sound effect while doing so. This will leave a blood trail. A corpse must be eaten for this to recharge. Starts out charged when demon is rolled.
    • Blood Stomp: Sister to garg stomp. Should slam the ground with a 5x5 radius, knocking down and doing 20 brute to all surrounding humanoids. Blood should be sprayed in this 5x5 radius. A corpse must be eaten for this to recharge. Starts out charged when demon is rolled.
    • Wherever the demon walks, blood should be left behind. This will be a passive.

**Honorary Mentions**:
    • Chaplain Weakness: When a demon is hit with a nullrod item, it’ll give a visible “weakened” status effect, obvious to both the demon and those attacking it. It’ll double all burn damage the demon takes, effect dissipates after 5 seconds.
    • Blood Eruption: I wanted to add this, but there’s problems with that ability from hemomancer, and it won’t be a good fit till I get it fixed. 
    • If killed, demon heart will be tied to the person who dealt the killing blow. Anyone else who attempts to eat it will get poisoned for 50 tox and then weakened, with text above similar to trying to use cult items: _The pulsating, blackened heart rejects your very essence!_ 

**Reasoning**:
> HP from 130 > 400.

This is needed because the demon will be out of blood for minimum of 20 seconds, likely getting swarmed by both assistants and lasergun fire.

> Damage remains the same, 30.
> Armor pen maxed, so an assistant will be killed just as fast as a miner.

This is better than raising the base damage. Assistants already die quick enough, its only armored people that sandbag the demon. Better off keeping time to kill the same for both parties than raising damage and having assistants die in three hits.

> Object damage raised high enough to break walls. Should also be able to break windows in one hit.

We don’t want a closed airlock or RCD wall to completely stop a chase, forcing the demon to twiddle thumbs for 20 seconds.

> Stacking damage resist and additional damage per hit on ckey controlled mob, 5% for both each hit, capped at 50%, starts lowering after 5 seconds without hitting anybody, should reach 0 stacks in 10 seconds.
> Every melee hit against ckey controlled mob heals 10 HP. They must be alive for it to heal.
> Every melee hit refreshes the temporary speed gained from surfacing from blood crawl, resetting the timer till it reverts to the normal, slower demon speed.

Values can be tweaked, but ideally we want to reward continued carnage in melee. Melee should be the demon absolute strong suit.

> Bloodcrawl cooldown is set to 20 seconds. 

The entire point of this design doc lies here. We remove hit and run, and we compensate for the demon having combat ability to survive in a bloodbath.

> Two second delay where you have to still to “channel” into blood. This ideally comes with a sprite effect to notify this is happening.
> In tandem with the above, demon becomes unable to be pushed or shuffled.

Both aspects here are needed. Half the frustration with current demon is the ability to vanish second things don’t go its way, leaving crew feel unsatisfied. This way, if its leaving you can tell, and you have your chance to kill it. If you fail to, its on you.

> Passive heal at the rate of 5% per second while in blood nullspace. This is capped at 200 HP.

Nobody wants to see the demon leave a fight barely alive, surface later, and get killed by an assistant. This will ensure a minimum level of survivability, in return for losing blood spots (the ones being cleaned).

> Dragging in corpses no longer works with pulling and eating one. Instead, when channeling is completed with Bloodcrawl, ALL corpses in a 3x3 square are taken into blood. 
> In tandem with the above, each ckey controlled corpse heals 100 HP, and increases max HP by 10. If four corpses or more are eaten at once, you get an additional 50 max HP increase, atop of the flat 10 values.

Rewards the demon for multi kills with eating them at once. Lesser benefit for only getting a single kill out of it. If you’re worried people can grab the corpse as its channeling and deny the heal, you can make the corpses lock in place. Personally, I’d see the corpse yoinking by crew to be good counterplay to keep.

> When Bloodcrawling out of nullspace, the blood within a 5x5 square begins to glow. The channeling will not be cancelled unless all blood within that square is cleaned. Any humanoids within the radius when the demon’s channeling is completed will be knocked down and start bleeding.

With the buffs demons received, the element of complete surprise needs to be axed. More than small red text is needed to notify people the demon is coming.

> Blood Charge: Smashes through walls, obstacles, and people, in a similar fashion to rod form. I imagine it having 7 tile range, unable to change direction once chosen, critting people if it hits them head on. Ideally, it could be given some sort of scream sound effect while doing so. This will leave a blood trail. A corpse must be eaten for this to recharge. Starts out charged when demon is rolled.

Demon will need a bit more oomph even with its HP and damage resist on hit, being that it can be kited, and has slowdown applied if it cant find anyone in melee to refresh. This will accomplish that. Can be given a normal recharge, but given it’ll crit people it hits, it should require a different sort of recharge mechanic imo.

> Blood Stomp: Sister to garg stomp. Should slam the ground with a 5x5 radius, knocking down and doing 20 brute to all surrounding humanoids. Blood should be sprayed in this 5x5 radius. A corpse must be eaten for this to recharge. Starts out charged when demon is rolled.

This one is the most needed addition to axing hit and run. If it getting surrounded and pummeled in melee, damage resist will not save it. Parrying will also be very effective, further reducing the demons heals. This will give the demon an edge up for a second wind in melee.

> Wherever the demon walks, blood should be left behind. This will be a passive.

Hit and run is no longer the mechanic. As such, it is a demon of murder, carnage, and blood, and bleeding where its dreaded heels walk is fair I feel.

> Chaplain Weakness: When a demon is hit with a nullrod item, it’ll give a visible “weakened” status effect, obvious to both the demon and those attacking it. It’ll double all burn damage the demon takes, effect dissipates after 5 seconds.

Values can be tweaked liberally, but chaplain needs more mechanics going for it. Chaplain is holy, demon is not, this weakness fits.

> Blood Eruption: I wanted to add this, but there’s problems with that ability from hemomancer, and it won’t be a good fit till it is fixed.

Sorry Hal, never brought this up sooner: Eruption is very mid. No notification the blood is coming, high damage, but its cooldown sucks, its overpriced, and its too short of range. To consider adding it to demon it should be updated for hemomancer first, ideally with longer range, lower cooldown and price, but a glowing blood effect to notify chasing officer they’re about to get their calves mauled. Should be a very short time period for officers to get out of the blood. A second at most.

> If killed, demon heart will be tied to the person who dealt the killing blow. Anyone else who attempts to eat it will get poisoned for 50 tox and then weakened, with text above similar to trying to use cult items: The pulsating, demonic heart rejects your very essence! 

Can be tossed. I don’t want to see the demon killed and the joy of the winner stolen by a slimey little greytider quicker on the alt click. Bonus points if you make the corpses from the demon launch out and deal brute to all objects and people nearby.

**Conclusion**:
That's a wrap. If everything goes how I imagined it and replayed it in my head, the demon will win if crew doesn’t arm up like a biohazard. If everyone has an IK, the demon will probably get some good kills but overall die. If the demon is bald, it’ll just die. 

In other words, if everything goes how I imagined it, the balance of it will remain the same EXCEPT that it’ll be funner to fight, and feel more rewarding to play.

Lastly, I made this with the expectation that this will never make it in and it’ll be never read by more than two people. My only reason for typing this all out is so when I next see “demon needs more than a simple tweak” or “demon just doesnt feel right” or “demon is badly designed” I’ll link this instead of watching them complain. 

Treat this as a framework, not a bible. If you like it, take it. If you like parts of it, take the parts you like and disregard the rest.

**Comment by Screampuff** [comment]
Great changes all around and feels far less of a cheesefest.
